itest/lnd_multi-hop_test.go

```diff
@@ -1790,7 +1954,13 @@ func createThreeHopNetwork(ht *lntest.HarnessTest,
 		go acceptChannel(ht.T, true, acceptStream)
 	}
 
+	var privateChan bool
+	if c == lnrpc.CommitmentType_SIMPLE_TAPROOT {
```

Comment:
If we wanna avoid creating the route hints manually, we could make the link `alice -> bob` public and keep `bob -> carol` as is,
```diff
diff --git a/itest/lnd_multi-hop_test.go b/itest/lnd_multi-hop_test.go
index 02c882735..8cea594ca 100644
--- a/itest/lnd_multi-hop_test.go
+++ b/itest/lnd_multi-hop_test.go
@@ -1948,14 +1948,16 @@ func createThreeHopNetwork(ht *lntest.HarnessTest,
 	}
 
 	var privateChan bool
+
+	aliceChanType := c
 	if c == lnrpc.CommitmentType_SIMPLE_TAPROOT {
 		privateChan = true
+		aliceChanType = lnrpc.CommitmentType_ANCHORS
 	}
 
 	aliceParams := lntest.OpenChannelParams{
-		Private:        privateChan,
 		Amt:            chanAmt,
-		CommitmentType: c,
+		CommitmentType: aliceChanType,
 		FundingShim:    aliceFundingShim,
 		ZeroConf:       zeroConf,
 	}
```
Non-blocking tho.